rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -78,12 +95,26 @@ public Start(Rsk rsk,
         this.rskSystemProperties = rskSystemProperties;
         this.web3Factory = web3Factory;
         this.repository = repository;
+        this.blockchain = blockchain;
+        this.channelManager = channelManager;
     }
 
     public void startNode(String[] args) throws Exception {
         logger.info("Starting RSK");
 
         CLIInterface.call(rskSystemProperties, args);
+        logger.info("Running {},  core version: {}-{}", rskSystemProperties.genesisInfo(), rskSystemProperties.projectVersion(), rskSystemProperties.projectVersionModifier());
```

Comment:
All of this is now logged under `start` instead of `general`, which makes so much sense!